packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/WorklistDataGrid.1/Examples.3/RowSelection.3.doc.mdx

```diff
@@ -1,3 +1,39 @@
 import RowSelection from './RowSelection?dev-site-example';
 
-<RowSelection title='Worklist Data Grid With Row Selection' />
+# Row Selection
+
+### Description
+The [Worklist DataGrid](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about) is built to operate in either of two selection modes:
+* Cell Selection (_default_)
+* **Row Selection**
+
+Row Selection Mode is controlled via a prop and is off by default. When creating a Worklist DataGrid that supports Row Selection, the prop has to be explicitly set by the Consumer. This example demonstrates a Worklist DataGrid with full support for row selection and can be referenced for [row centric functionalities](http://localhost:8080/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/row-selection#implemented-functionality).
```

Comment:
```suggestion
Row Selection Mode is controlled via a prop and is `false` by default.
When creating a Worklist DataGrid that supports Row Selection, the prop has to be explicitly set.
This example demonstrates a WorklistDataGrid with full support for row selection and can be referenced for [row centric functionalities](http://localhost:8080/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/examples/row-selection#implemented-functionality).
```

Reply:
Changed sentences in the paragraph to new line in previous commit. I did not take the _false_ keyword suggestion as describing a feature as _on/off_ makes more sense compared to _true/false_.